pkg/parse/annotations.go

```diff
@@ -50,9 +50,9 @@ func addAnnotationsAndLabels(objs []ast.FileObject, scope declared.Scope, syncNa
 		core.SetAnnotation(obj, metadata.ResourceIDKey, core.GKNN(obj))
 		core.SetAnnotation(obj, metadata.OwningInventoryKey, inventoryID)
 
-		value := core.GetAnnotation(obj, metadata.ResourceManagementKey)
-		if value != metadata.ResourceManagementDisabled {
-			core.SetAnnotation(obj, metadata.ResourceManagementKey, metadata.ResourceManagementEnabled)
+		value := core.GetAnnotation(obj, metadata.ManagementModeAnnotationKey)
+		if value != metadata.ManagementDisabled.String() {
```

Comment:
nit: use `!metadata.IsManagementDisabled`

the line below could also use `WithManagementMode`

Reply:
WithManagementMode constructs a function, which makes it awkward to use in this case. So I left it as-is.

I suppose we could add a SetManagementMode...